tests/cpydiff/builtin_next_arg2.py

```diff
@@ -1,6 +1,6 @@
 """
 categories: Modules,builtins
-description: Second argument to next() is not implemented
+description: Second argument to next() is optional
```

Comment:
The nature of this change is unclear. Yes, second argument to next() is optional. How that is difference to CPython? If this change is merged, this file would need to go completely. (Assuming that  tests/cpydiff/ track differences between the "maximal" MicroPython config and CPython.)

Reply:
>  track differences between the "maximal"

Ok thanks for clearing that out.